composer.json

```diff
@@ -30,12 +30,10 @@
         "phpunit/PHPUnit": "~4.0"
     },
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
-        "zendframework/zend-db": "Zend\\Db component",
```

Comment:
zend-db needs to be here; one of the possible log writers is for DB, and that uses a zend-db adapter.